github-orgs/kubeflow/org.yaml

```diff
@@ -1132,6 +1134,7 @@ orgs:
             - andreyvelich
             - ChenYi015
             - ckadner
+            - dhirajsb
```

Comment:
Please check the process to add contributor to the WG Leads: https://www.kubeflow.org/docs/about/membership/
Usually, adding person to the WG leads happen after adding them to members, reviewers, and approvers.
cc @kubeflow/wg-data-leads

Reply:
Thanks for the reminder @andreyvelich . Changing the PR to add membership for now and will request addition to wg separately.